tools/simhostroute.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -x
+
+# Helper script to set up host route to NuttX simulator
+# and set up IP Tables to allow it to access the 
+# internet.
+#
+# Note that on Linux you may have to set kernel capabilities
+# on the nuttx executable to allow NuttX to access the tap device:
+#
+# sudo setcap cap_net_admin+ep ./nuttx
+
+if [ $# != 2 ]; then
+  echo "Usage: $0 <interface> <on|off>"
+  exit 1
+fi
+
+IF_HOST=$1
+STATUS=$2
+
+IF_BRIDGE=nuttx0
+IP_NET="10.0.1.0/24"
+IP_NETMASK="255.255.255.0"
+IP_BROADCAST="10.0.0.255"
+IP_HOST="10.0.1.1"
+IP_NUTTX="10.0.1.2"
+
+if [ "$STATUS" == "on" ]; then
+    ip link add $IF_BRIDGE type bridge
```

Comment:
It seem that we are still using bridge mode? does this script work with CONFIG_SIM_NET_HOST_ROUTE?

Reply:
@xiaoxiang781216 It's using host route mode, and should have CONFIG_SIM_NET_HOST_ROUTE set. I'll look at my setup tomorrow and redo the defconfig so it has that.

As for the name `$IF_BRIDGE` it should really be called `$IF_TAP` or `$IF_NUTTX`. I'll fix that too.